VideoApi/Testing.Common/Assertions/AssertConferenceDetailsResponse.cs

```diff
@@ -46,7 +46,6 @@ public static void ForConference(ConferenceDetailsResponse conference)
             conference.MeetingRoom.JudgeUri.Should().NotBeNull();
             conference.MeetingRoom.ParticipantUri.Should().NotBeNull();
             conference.MeetingRoom.PexipNode.Should().NotBeNull();
-            conference.MeetingRoom.TelephoneConferenceId.Should().NotBeNull();
```

Comment:
Has this become optional? I thought every conference has a telephone conference id

Reply:
I believe, it will be optional before Kinly date for confirmed conferences passed

Reply:
I double-checked the code and booking a new conference successfully includes going to Kinly API to get the meeting room details. The stub reflects this behaviour also.
We only need to hide the value in the UI, not in the API.

Reply:
\done, reversed this change